WRP-10826: Fix `WizardPanels` to restore focus properly after a transition (#1425)
enactjs/sandstone: CHANGELOG.md

The following is a curated list of changes in the Enact sandstone module, newest changes on the top.

## [unreleased]

### Fixed

> - `sandstone/WizardPanels` to handle focus properly when switching views in Chrome version 104 or higher

Reviewer: ```suggestion
- `sandstone/WizardPanels` to restore focus properly after a transition in Chrome 104 or higher
```
I would like to discuss how to update the changelog with merger @0x64.
